igrac/models.py

```diff
@@ -1,3 +1,13 @@
 from django.db import models
+from django.contrib.auth.signals import user_logged_in
```

Comment:
hi @zamuzakki can you move the changes in this file into igrac/utilities.py (create this new file) ?

Reply:
Hi @ann26 , I've moved the changes to igrac/signals.py.

Reply:
thanks @zamuzakki